docs/www/rpk-commands.md

```diff
@@ -173,6 +173,39 @@ Flags:
       --watermarks      If enabled, will display the topic's partitions' high watermarks (default: true)
 ```
 
+### topic produce
+
+Produce a record. Reads data from stdin.
+
+```cmd
+Usage:
+  rpk topic produce <topic> [flags]
+
+Flags:
+  -H, --header stringArray   Header in format <key>:<value>. May be used multiple times to add more headers.
+  -j, --jvm-partitioner      Use a JVM-compatible partitioner. If --partition is passed with a positive value, this will be overridden and a manual partitioner will be used.
+  -k, --key string           Key for the record. Currently only strings are supported.
+  -n, --num int              Number of records to send. (default 1)
+  -p, --partition int32      Partition to produce to. (default -1)
```

Comment:
s/default/Default:/

Reply:
This is actual output from `--help`, so even though it might be capitalized here, the command's output won't be. Should I still change it?

Reply:
Ideally we would change it in the help output and pull that into the docs.